frontend/src/components/GroupList.tsx

```diff
@@ -81,15 +82,22 @@ export default function GroupList(props: { session: Session | null }) {
       },
     };
 
-    const p1 = API.getAcceptedGroups(requestOptions)
+    const p1 = API.getGroups(requestOptions, true)
```

Comment:
instead of having two requests here, we can have the API return all the groups. Then in the frontend here with javascript, filter by active & inactive groups.